odmantic/index.py

```diff
@@ -13,8 +13,7 @@ def __init__(self, unique: bool, index_name: Optional[str]) -> None:
         self.index_name = index_name
 
     @abstractmethod
-    def get_index_specifier(self) -> Sequence[Tuple[str, int]]:
-        ...
+    def get_index_specifier(self) -> Sequence[Tuple[str, int]]: ...
```

Comment:
I'm not sure what creates all those changes with the ellipsis. Maybe ruff/black/isort inconsistencies?